### ISSUE TITLE
Requirement Gathering

### ISSUE THREAD

**Issue by CodeLawd** [playproject-io/contracting#39]
## `@todo`
- [ ] **User Requirement**
     - [ ] User Research
          - [ ] User Interview (3 days)
                 - `@input` :question: `Prepare user interview questions`
                 - `@output` :question: `Document containing user interview result`
          - [ ] Stakeholder's Interview (1 day)
                - `@input` :question: `Prepare stakeholder interview questions`
                - `@output` :question: `Document containing stakeholder interview result`
          - [ ] Persona Creation (1 day)
                - `@input` :question: `Document containing interview result`
                - `@output` :question: `User persona document`
          - [ ] User Stories Map (1 day)
                - `@input` :question: `Document containing interview result`
                - `@output` :question: `User story map document`
          - [ ] User Task Flow (1 day)
                - `@input` :question: `User story map document`
                - `@output` :question: `Task flow document`
- [ ] **Product Requirement (2 days)**
     - [ ] Study previous features and functionalities from the input document/resources
              - `@input` :question: [website section for organizing structure and processes](https://user-images.githubusercontent.com/1076427/145653435-34a1ad82-0512-4c20-84eb-317681bd4197.png)
              - `@input` :question: [dat-ecosystem garden application form](https://docs.google.com/forms/d/e/1FAIpQLSf9YxzVOTdPQKTwFoHzPTpSBNEm9dqOdYnZdP_bPAy4hrGQ0A/viewform)
              - `@input` :question: [Joining Rules](https://hackmd.io/5zL2Qt4_Sv6-_YgY8rJspg)
              - `@input` :question: [dat-ecosystem open collective account](https://opencollective.com/dat)
              - `@output` :question: `Document containing layout structures, colors palletes, navigation structure and images`
- [ ] **Business Requirement(2 days)**
   - [ ]    Conduct Comepetitive Analysis
      - [ ] `Identify competitors`
      - [ ] `Analyze competitors' online presence`
      - [ ] `Check online reviews`
      - [ ] `Talk to competitors’ customers`
      - [ ] `Identify their strengths and weaknesses`

`@output` :question: `dat ecosystem requirement document`
 - Next #40 

## `@info`

The `product requirement` section defines the product we are about to build: It outlines the product's purpose, its features, functionalities, and behavior. Next, you share the PRD with (and seek input from) stakeholders - business and technical teams who will help build, launch or market your product.

A `competitive analysis` is a strategy that involves researching major competitors to gain insight into their products, sales, and marketing tactics. Implementing stronger business strategies, warding off competitors, and capturing market share are just a few benefits of conducting a competitive market analysis.

**Comment by serapath** [comment]
**feedback `2022.04.28`**

good, but :-)

what we don't do is linking generically to the sequence in which todos are meant to be done, that is implicitely given by the inputs required to start todos ...because often times things can also be done in parallel and the only constraint is the input/output structure so no generic next or from links needed if they are not attached to inputs or outputs


1. the first todo `requirement gathering` does not need to be a link to itself, raw text is enough

2. the inputs should not have a checkbox and the reason is, that it is good enough to know there is an input planned and once you created that input as an output elsewhere (e.g. in #45), then you can update the symbol from :question: to :package: and you are also familiar with the input ...but if the input was created by somebody else than you, then just make an extra todo with a checkbox that says:
```
* [ ] study User Research
    * `@input` :question: [`user research document`](#...)
```

also ...no input or output should ever link to issues, they always have to link to documents and only if inputs were produced in other issues you should use `from #<no>` to indicate that issue and if an output is used in other issues you can list them as a sub bullet point list of `next #<no>` to iterate all the issues the output is used in as an input

3. lastly, the `next #40` needs to be attached to at least a single output if you want to do this issue at all :-)

**Comment by serapath** [comment]
merged into #49
